_opensearch/install/docker-upgrade.md

```diff
@@ -0,0 +1,211 @@
+```
+**Sample response:**
+```bash
+{
+  "_shards" : {
+    "total" : 4,
+    "successful" : 4,
+    "failed" : 0
+  }
+}
+```
+1. Disable shard replication to prevent shard replicas from being created while nodes are being taken offline.
+```bash
+curl -X PUT "http://localhost:9200/_cluster/settings?pretty" -H 'Content-type: application/json' -d'{"persistent":{"cluster.routing.allocation.enable":"primaries"}}'
+```
+**Sample response:**
+```bash
+{
+  "acknowledged" : true,
+  "persistent" : {
+    "cluster" : {
+      "routing" : {
+        "allocation" : {
+          "enable" : "primaries"
+        }
+      }
+    }
+  },
+  "transient" : { }
+}
+```
+1. Verify the health of your OpenSearch cluster.
+```bash
+curl "http://localhost:9200/_cluster/health?pretty"
+```
+A status of **green** indicates that all primary and replica shards are allocated.
+**Sample response:**
+```bash
+{
+  "cluster_name" : "opensearch-dev-cluster",
+  "status" : "green",
+  "timed_out" : false,
+  "number_of_nodes" : 4,
+  "number_of_data_nodes" : 4,
+  "discovered_master" : true,
+  "active_primary_shards" : 2,
+  "active_shards" : 4,
+  "relocating_shards" : 0,
+  "initializing_shards" : 0,
+  "unassigned_shards" : 0,
+  "delayed_unassigned_shards" : 0,
+  "number_of_pending_tasks" : 0,
+  "number_of_in_flight_fetch" : 0,
+  "task_max_waiting_in_queue_millis" : 0,
+  "active_shards_percent_as_number" : 100.0
+}
+```
+1. List the running containers in your OpenSearch cluster and make a note of the container IDs and names.
+```bash
+docker container ls
+```
+**Sample output:**
+```bash
+CONTAINER ID   IMAGE                                           COMMAND                  CREATED         STATUS         PORTS                                                                NAMES
+af1ec1c3fef7   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   4 minutes ago   Up 4 minutes   9300/tcp, 9650/tcp, 0.0.0.0:9200->9200/tcp, 0.0.0.0:9600->9600/tcp   os-node-04
+1d47c0da60ad   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   4 minutes ago   Up 4 minutes   9300/tcp, 9650/tcp, 0.0.0.0:9200->9200/tcp, 0.0.0.0:9600->9600/tcp   os-node-03
+f553b5ec870b   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   4 minutes ago   Up 4 minutes   9300/tcp, 9650/tcp, 0.0.0.0:9200->9200/tcp, 0.0.0.0:9600->9600/tcp   os-node-02
+934e4325d9a4   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   4 minutes ago   Up 4 minutes   9300/tcp, 9650/tcp, 0.0.0.0:9200->9200/tcp, 0.0.0.0:9600->9600/tcp   os-node-01
+```
+1. Query `_cat/nodes` to determine which node is operating as the cluster manager. Note that OpenSearch 1.x versions use the heading "master," which has been deprecated and replaced by "cluster_manager" in OpenSearch 2.x versions.
+```bash
+curl -s "http://localhost:9200/_cat/nodes?v&h=name,version,node.role,master" | column -t
+```
+**Sample output:**
+```bash
+name        version  node.role  master
```

Comment:
It wasn't clear to me when reading this chart whether this list contained only master eligible nodes with the master denoted with * or if this simply contained all nodes (possibly not master eligible) with the master denoted with *

Reply:
Yeah good call out. I'll add commentary for outputs like that. "The response indicates that `os-node-02` is the cluster manager" or something